design/admin/stylesheets/pagelayout.css

```diff
@@ -472,10 +472,11 @@ ul.leftmenu-items li.current a
 #widthcontrol-handler
 {
     position: absolute;
-    right: 20px;
-    top: 0;
+    right: -30px;
+    top:-20px;
     bottom: 0; /* strech div to full width of relative box */
     width: 10px;
+    height: 5000px;
```

Comment:
Are these changes really relevant here?

Reply:
@andrerom good catch, probably came from somewhere else, going to update this PR during the weeekend and remove those changes.

Reply:
😃